File: src/backend/NHP_Domain/Entities/Image.cs
namespace NHP_Domain.Entities;

public class Image
{
    public long Id { get; set; }
    public String UrlImage { get; set; }
    public String NomImage { get; set; }
    
    // OneToMany
    //public virtual List<Logement>? Logements { get; set; }

    public override string ToString()
    {
        return $"urlImage: {UrlImage} " +
               $"nomImage: {NomImage} ";
    }
}

File: src/frontend/src/domain/entities/Image.ts
import type { ILogement } from '@/domain/entities/Logement';

export interface IImage {
    id: number;
    urlImage: string;
    nomImage: string;
    logements?: ILogement[]; // Relation OneToMany
}

export class Image implements IImage {
    constructor(
        public id: number,
        public urlImage: string,
        public nomImage: string,
        public logements?: ILogement[]
    ) {}
}